git2consul/wp-config.json

```diff
@@ -0,0 +1,20 @@
+{
+	"config": [
+		{ "site_url": "http://my-site.example.com" },
+
```

Comment:
We also need details to support bootstrapping the site. See https://github.com/misterbisson/triton-wordpress/pull/1#discussion_r50655806

These include:

- site title
- admin username
- admin password
- admin email address